app/controllers/search_controller.rb

```diff
@@ -15,7 +15,7 @@ def show
 
     puts query_string
 
-    @papers = Paper.where(query_string)
+    @results = EvidenceSource.where(query_string)
```

Comment:
I found there are more tables other team uploaded ,for example evidence_source_authors and evidence_source_ratings.
we should consider them.